frameworks/elastic/tests/test_kibana.py

```diff
@@ -42,6 +43,37 @@ def test_config_with_custom_yml() -> None:
         _uninstall_services()
 
 
+@pytest.mark.sanity
+def test_config_with_custom_placement():
+
+    try:
+        _uninstall_services()
+
+        non_default_placement = [["hostname", "CLUSTER"]]
+
+        sdk_install.install(
+            config.PACKAGE_NAME,
+            service_name=config.SERVICE_NAME,
+            expected_running_tasks=config.DEFAULT_TASK_COUNT,
+        )
+
+        sdk_install.install(
+            config.KIBANA_PACKAGE_NAME,
+            config.KIBANA_SERVICE_NAME,
+            0,
+            {"kibana": {"placement": non_default_placement}},
+            wait_for_deployment=False,
+            insert_strict_options=False,
+        )
+
+        marathon_constraints = sdk_marathon.get_config(config.KIBANA_SERVICE_NAME)["constraints"]
+
+        assert marathon_constraints == non_default_placement
```

Comment:
In addition to checking that the constraints match, let's also check that the Admin Router endpoint works with `config.check_kibana_adminrouter_integration`. That's what tells us that Kibana is _actually_ working.

Reply:
something like this?
```python
assert config.check_kibana_adminrouter_integration("service/{}/".format(config.KIBANA_SERVICE_NAME))
assert config.check_kibana_adminrouter_integration("service/{}/app/kibana".format(config.KIBANA_SERVICE_NAME))
```

Reply:
Yep. The first line should be sufficient.